.github/workflows/ci-docker.yml

```diff
@@ -316,7 +316,7 @@ jobs:
         id: buildx
 
       - name: Cache Docker Layers
-        uses: actions/cache@v3
+        uses: actions/cache@v4
```

Comment:
Please revert this. This file is managed in out `.github` repo.

Reply:
For the docs, you can enter this for the capability `50;52;60;61;62;70;75;80;86;90`